### PR TITLE
Add pypy38/pypy39

### PR DESCRIPTION
**Pull request by h-vetinari**
For https://github.com/conda-forge/pypy-meta-feedstock/pull/22

CC @isuruf @mattip

### CODE REVIEW

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -5,6 +5,8 @@ python_abi_tag:
  - cp310
  - pypy36_pp73  # [not win64]
  - pypy37_pp73
```
Comment (isuruf) [inline recipe/conda_build_config.yaml]: Can you drop these two?

Comment (h-vetinari) [reply recipe/conda_build_config.yaml]: Done.

**Comment by h-vetinari** [comment]
Do we need to bump the build-number?

**Comment by isuruf** [comment]
Nope. We don't use hashes in this recipe.

**Comment by isuruf** [comment]
Thanks for the PR

**Pull request merged**